changelogs/fragments/611-argcomplete.yml

```diff
@@ -0,0 +1,5 @@
+minor_changes:
+  - "If you are using `argcomplete <https://pypi.org/project/argcomplete/>`__, you can now tab-complete ``antsibull-build`` command lines.
```

Comment:
https://github.com/ansible-community/antsibull-changelog/pull/173/files#r1720762631

Reply:
That doesn't apply since `argcomplete` wasn't there before.